packages/server/src/resolvers.ts

```diff
@@ -119,7 +119,7 @@ export const resolvers = {
         }
       })
     },
-    createLocation: async (locationInput: Prisma.LocationCreateInput, context: Context) => {
+    /*createLocation: async (locationInput: Prisma.LocationCreateInput, context: Context) => {
```

Comment:
Why is this commented out?

Reply:
Because it doesn't work and aman commented it out